src/components/layout/Header.test.js

```diff
@@ -0,0 +1,9 @@
+import { render } from '@testing-library/react'
+import Header from './Header'
+
+describe('Header', () => {
+    it('reders correctly', () => {
```

Comment:
props to you fro testing! 👌🎉
still on my to do list to implement that ;)

Reply:
Thank, you :D You will handle this as well :)